fem/post/create_res_sim.py

```diff
@@ -182,12 +187,18 @@ def __read_cli():
                      help="read in disp.dat file that has node IDs saved for"
                           "each timestep",
                      action="store_true")
+    par.add_argument("--plane_pos", 
+                     help = "pos of plane wanted to extract",
+                     default = 0.0)
+    par.add_argument("--plane_orientation",
+                     help = "what orientation plane to use 0 = elev, 1 = lat, 2 = ax",
+                     default = 0)
     args = par.parse_args()
 
     return args
 
 
-def extract_image_plane(snic, axes, ele_pos):
+def extract_image_plane(snic, axes, plane_pos, direction = 0):
```

Comment:
We should check (with a test) if the old syntax using the `ele_pos` keyword argument will still work if someone has old code that doesn't utilize these new keyword arguments.  If not, we'll increment up the major version number of the repo to reflect the API change.